core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -21,6 +21,9 @@ final case class Kleisli[F[_], A, B](run: A => F[B]) { self =>
   def map[C](f: B => C)(implicit F: Functor[F]): Kleisli[F, A, C] =
     Kleisli(a => F.map(run(a))(f))
 
+  def contramap[C](f: C => B)(implicit F: Contravariant[F]): Kleisli[F, A, C] =
+    Kleisli(a => F.contramap(run(a))(f))
+
```

Comment:
I think the `contramap` that I would expect for `Kleisli` would be the `local` method that already exists on it. I think in practice the `F` in a `Kleisli` is often going to be covariant (not contravariant) in nature (`Future`, `Task`, `Xor`, etc), so I don't know how often this would be helpful. If you have some good examples of how you might use this, I'd definitely be interested in them.

Otherwise, should we get rid of this one and use `local` for the `Contravariant` instance?